contracts/container/contract.go

```diff
@@ -491,6 +494,67 @@ func List(owner []byte) [][]byte {
 	return list
 }
 
+// AddNextEpochNodes adds container members for the next epoch. Changes must be
+// committed with [CommitContainerListUpdate]. Without submitting, it is a no-op
```

Comment:
i know that u mean, but `no-op` can cause misunderstaning: contract storage is changed, GAS spent, although user sees no side effect through the API. I'd say `AddNextEpochNodes accumulates passed nodes as container members for the next epoch to be committed using [CommitContainerListUpdate].`

Reply:
i meant without submitting it is not possible to see any changes via public interfaces, it is an alphabet-only method, so gas and any other things are known and does not mean a lot. but don't mind your way